app/assets/stylesheets/thredded/components/_post.scss

```diff
@@ -4,6 +4,71 @@
   @include thredded-media-mobile {
     margin-bottom: $thredded-base-spacing;
   }
+  .thredded--post--dropbtn .thredded--post--three-dot-menu {
+    fill: currentColor;
+    color: $thredded-action-color;
+    display: inline-block;
+    font-size: 1em;
+    width: 1rem;
+    height: 1rem;
+    padding-left: 1.5rem;
+  }
+
+  .thredded--post--dropdown {
+    position: relative;
+    display: inline-block;
+    float: right;
+    text-align: right;
+    cursor: pointer;
+    @media print {
+      display: none;
+    }
+  }
+
+  .thredded--post--actions {
+    display: none;
+    position: absolute;
+    right: 0;
+    background-color: #f9f9f9;
+    min-width: 170px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+    cursor: pointer;
+  }
+
+  .thredded--post--actions a {
+    font-size: $thredded-font-size-small;
+    color: $thredded-text-color;
+    //color: $thredded-action-color;
+    text-decoration: none;
+    //transition: $thredded-action-transition;
+    padding: 5px 10px;
```

Comment:
Let's use the Material dimensions from here: https://material.io/guidelines/components/menus.html#menus-specs